src/main/java/com/opentable/versionedconfig/GitService.java

```diff
@@ -46,20 +47,20 @@
     private static final Logger LOG = LoggerFactory.getLogger(GitService.class);
 
     private final Path checkoutDirectory;
-    private final GitProperties serviceConfig;
+    private final GitProperties config;
 
     private final GitOperations gitOperations;
 
     private AtomicReference<ObjectId> latestKnownObjectId;
 
     @Inject
-    GitService(GitProperties serviceConfig) throws VersioningServiceException {
-        this.serviceConfig = serviceConfig;
+    GitService(GitProperties config) throws VersioningServiceException {
+        this.config = config;
         this.checkoutDirectory = getCheckoutPath();
         try {
-            this.gitOperations = new GitOperations(serviceConfig, checkoutDirectory);
+            this.gitOperations = new GitOperations(config, checkoutDirectory);
 
-            gitOperations.checkoutBranch(serviceConfig.getBranch());
+            gitOperations.checkoutBranch(config.getBranch());
             this.latestKnownObjectId = new AtomicReference<>(ObjectId.zeroId());
             LOG.info("Initializing {}, next update = {}", checkoutDirectory, latestKnownObjectId.get());
```

Comment:
Since latestKnownOjectId always == zeroID (unless I'm missing something), this partially seems useless?